xsnippet/api/middlewares/auth.py

```diff
@@ -19,19 +19,17 @@
 SECRET_LEN = 64
 
 
-async def setup(app):
+async def setup(app, conf):
```

Comment:
nit: we no longer need the `app` argument

Reply:
That's right, however, it's passed by `aiohttp` anyway, so we can't just ignore it. :(